p2p/protocols/accounting.go

```diff
@@ -27,9 +27,9 @@ type PriceOracle interface {
 
 type BalanceManager interface {
 	//Credit is crediting the peer, charging local node
-	Credit(peer *Peer, amount uint64) error
+	Credit(peer *Peer, amount uint64, size uint32) error
```

Comment:
i dont like this , this layer should not know about any size or message

Reply:
you meter sizes where sizes are handled, in accounting. 
The module that credits and debits, knows nothing about sizes and should just handle balances. No?